### DISCUSSION TITLE
Would this work with the 8440P?

### DISCUSSION THREAD

**Discussion by dolence** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#59] [General]
I'm trying to install to an old 8440P. Would this work?

**Comment by chris1111** [comment]
8440P is not support because its a first gen Graphics Intel HD you need a 7 series Intel HD 4000

**Reply by dolence** [reply]
Which is the latest version of OS X that would run on it? My 8440 have an nvidia quadro. Wouldn't it run on it?
